metadata.rb

```diff
@@ -9,3 +9,5 @@
 source_url       'https://github.com/chef-cookbooks/windows_dfs'
 issues_url       'https://github.com/chef-cookbooks/windows_dfs/issues'
 chef_version     '>= 13.0'
+
+depends 'windows', '>=5.2'
```

Comment:
The version bit doesn't actually do anything

```suggestion
 depends 'windows'
```